cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -74,128 +71,130 @@ function yupStringRequired(errorMessage: string) {
     return yup.string().nullable().required(errorMessage);
 }
 
-function AwsClusterSettingsStep(props: Partial<MCSettings>) {
-    const { handleValueChange, currentStep, deploy, defaultData, message } = props;
-    const { awsState } = useContext(AwsStore);
+function AwsClusterSettingsStep(props: Partial<StepProps>) {
+    const { updateTabStatus, currentStep, deploy } = props;
+    const { awsState, awsDispatch } = useContext(AwsStore);
     const awsClusterSettingsFormSchema = yup.object(createYupSchemaObject()).required();
     const methods = useForm<AwsClusterSettingFormInputs>({
         resolver: yupResolver(awsClusterSettingsFormSchema),
+        mode: 'all',
     });
     const [notification, setNotification] = useState<Notification | null>(null);
     const [images, setImages] = useState<AWSVirtualMachine[]>([]);
     const {
         handleSubmit,
         formState: { errors },
-        register,
         setValue,
     } = methods;
 
     function dismissAlert() {
         setNotification(null);
     }
 
-    const setImageParameters = (image) => {
-        if (handleValueChange) {
-            handleValueChange(INPUT_CHANGE, 'IMAGE_INFO', image, currentStep, errors);
-        }
-    };
+    // update tab status bar
+    if (updateTabStatus) {
+        UseUpdateTabStatus(errors, currentStep, updateTabStatus);
+    }
 
     const handleMCCreation = () => {
         if (deploy) {
             deploy();
         }
     };
-
+    const region = awsState[STORE_SECTION_FORM].REGION;
     useEffect(() => {
-        try {
-            AwsService.getAwsosImages(awsState[STORE_SECTION_FORM].REGION).then((data) => {
+        const setImageParameters = (image: any) => {
```

Comment:
nit: I was thinking of `setImage` rather than `setImageParameters`, but given that the field name is IMAGE_INFO, maybe `setImageInfo`?

Reply:
renamed